src/compiler/irgen.jl

```diff
@@ -4,17 +4,30 @@ function module_setup(mod::LLVM.Module)
     triple!(mod, "amdgcn-amd-amdhsa")
 
     # add debug info metadata
-    push!(metadata(mod), "llvm.module.flags",
-         MDNode([ConstantInt(Int32(1), JuliaContext()),    # llvm::Module::Error
-                 MDString("Debug Info Version"),
-                 ConstantInt(DEBUG_METADATA_VERSION(), JuliaContext())]))
+        push!(metadata(mod), "llvm.module.flags",
+             MDNode([ConstantInt(Int32(1), JuliaContext()),    # llvm::Module::Error
+                     MDString("Debug Info Version"),
+                     ConstantInt(DEBUG_METADATA_VERSION(), JuliaContext())]))    
 end
 
+# make function names safe for GCN -- maybe not necessary (ptx derived change)
```

Comment:
I seem to recall that we didn't actually need to do this; @vchuravy do you recall?

Reply:
Doesn't hurt to do it, but if we don't need it we don't need it.

Reply:
I recall testing with some absurd Unicode characters and not having any issues, so I'll drop this code.